pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/stream/WritableStreamingData.java

```diff
@@ -100,11 +101,13 @@ public boolean hasRemaining() {
      * @return the actual number of bytes skipped.
      */
     @Override
-    public long skip(long count) {
+    public long skip(final long count) {
         try {
-            // We can only skip UP TO count, but if there are fewer bytes remaining, then we can only skip that many.
+            // We can only skip UP TO count.
             // And if the maximum bytes we can end up skipping is not positive, then we can't skip any bytes.
-            count = Math.min(count, remaining());
```

Comment:
As with `ReadableSequentialData`, the current behavior is specified in `SequentialData.skip()` JavaDoc. It states that if the requested number of bytes is past the limit, bytes are skipped only up to the limit, but no exceptions are thrown. If we're going to change this and throw an exception, `SequentialData.skip()` JavaDoc should be updated.

Reply:
Yep, they both inherit from the SequentialData, so I've just updated that javadoc already.